pkg/api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
 /*
```

Comment:
What is this file?

Reply:
this file is auto generated by kube builder: https://book.kubebuilder.io/cronjob-tutorial/other-api-files.html#zz_generateddeepcopygo